package.json

```diff
@@ -37,16 +37,16 @@
     "preinstall": "node ./preinstall_check",
     "kbn": "node scripts/kbn",
     "es": "node scripts/es",
-    "elasticsearch": "echo 'use `yarn es snapshot -E path.data=../data/`'",
+    "elasticsearch": "node scripts/es snapshot -E path.data=../data/",
```

Comment:
Lets either remove this command, or leave it how it was. This was added to provide similar compatibility to what was prior to kbn-es, but it should not be how folks are running ES.